Remove third_party/llvm-project/libcxx{,abi}

niranjanyardi opened the pull request:
Fixed: 391403083

niranjanyardi commented:
@andrewsavage1 I'll added the reviewers you added once the pre-submits pass. I removed reviewers and later realized you had added them.

dahlstrom-g commented:
> > @andrewsavage1 I'll added the reviewers you added once the pre-submits pass. I removed reviewers and later realized you had added them.
> 
> Hmm, i added dana as a reviewer and pinged him a couple of times last 2 days with no response. I'm adding brian as a 3P reviewer

I've tried a few times to send a review
but the GitHub PR Web UI just hangs/crashes. 😠

I'm wondering if we can remove `third_party/llvm-project/libunwind`
added by commit d2c87a49 since it's also included here:
https://github.com/youtube/cobalt/blob/e976d0f7/DEPS#L548-L551

The commit message (and PR description) has various
superficial and other minor defects; I'd replace it with just

```
Remove third_party/llvm-project/libcxx{,abi}

Fixed: 391403083
```

niranjanyardi commented:
> > > @andrewsavage1 I'll added the reviewers you added once the pre-submits pass. I removed reviewers and later realized you had added them.
> > 
> > 
> > Hmm, i added dana as a reviewer and pinged him a couple of times last 2 days with no response. I'm adding brian as a 3P reviewer
> 
> I've tried a few times to send a review but the GitHub PR Web UI just hangs/crashes. 😠
> 
> I'm wondering if we can remove `third_party/llvm-project/libunwind` added by commit [d2c87a4](https://github.com/youtube/cobalt/commit/d2c87a49d3e83175ce6fe9766047bd3ddc5ba093) since it's also included here: [`e976d0f7`/DEPS#L548-L551](https://github.com/youtube/cobalt/blob/e976d0f7/DEPS#L548-L551)
> 
> The commit message (and PR description) has various superficial and other minor defects; I'd replace it with just
> 
> ```
> Remove third_party/llvm-project/libcxx{,abi}
> 
> Fixed: 391403083
> ```

Not sure if it's needed later. I can remove it later once I have confidence it's not needed .
 I don't think it's being used right now : so should be safe to remove. I'm not sure why it was added in the first place
